sycl/test/on-device/defaultcontext/default.cpp

```diff
@@ -0,0 +1,30 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
Shouldn't  the on-device tests be moved to https://github.com/intel/llvm-test-suite? There are contained E2E tests. please contact @vladimirlaz  on the details

Reply:
My understanding was that they go here first.